alpine-openjdk/VERSIONS.md

```diff
@@ -0,0 +1,32 @@
+# alpine-openjdk
+
+This file contains all software versions, that correspond to a version of this image itself. Read more about the [alpine-openjdk image here][alpineopenjdk].
+
+It should be noted that the versions used for OpenJDK follow the convention found found in [JEP 223][openjdk-jep223]
+
+## Latest
+
+Same as v8.92.14.
+
+Usage: `smebberson/alpine-openjdk` or `smebberson/alpine-openjdk:latest`.
+
+## v8.92.14
+
+- [smebberson/alpine-base: v3.0.0][smebbersonalpinebase300]
```

Comment:
As mentioned above, there is a new version of `alpine-base` that you can and should pin these images to.